app/public/js/isWrapped.js

```diff
@@ -0,0 +1,11 @@
+(function (global) {
+  'use strict';
+  var $ = global.jQuery;
+  $('h2.results__name').each(function(){
+    var $height = $(this).height();
```

Comment:
The convention to prepend variable names with `$` is only applicable when that variable is a jQuery element. It can be removed in this instance as height is just a number.